src/shared/components/VideoTileBase/VideoTileBase.tsx

```diff
@@ -3,6 +3,7 @@ import { CSSTransition, SwitchTransition } from 'react-transition-group'
 import useResizeObserver from 'use-resize-observer'
 
 import { useContextMenu } from '@/hooks/useContextMenu'
+import { UploadStatus } from '@/providers/uploadsManager/types'
```

Comment:
This will likely introduce chunking issues as well. I guess we could elevate this type to some global `src/types/` or something alike.